website/docs/cdktf/cli-reference/commands.mdx

```diff
@@ -109,13 +109,7 @@ Examples:
 Convert a local file.
 
 ```bash
-cat main.tf | cdktf convert > imported.ts`
-```
-
-Convert HCL in your clipboard to Python on OSX.
-
-```bash
-pbpaste | cdktf convert --language python | pbcopy
```

Comment:
It has an extra --language python attribute though 🤔  and also pastes into pbcopy

Reply:
It's the next example (line 124ish) that's identical to this one.

Reply:
Ah I see 👍